webui/react/src/components/UserAvatar.tsx

```diff
@@ -10,16 +9,8 @@ export interface Props extends Omit<AvatarProps, 'darkLight' | 'displayName'> {
 }
 
 const UserAvatar: React.FC<Props> = ({ userId, ...rest }) => {
-  const [ displayName, setDisplayName ] = useState('');
   const { ui, users } = useStore();
-  const fetchUsers = useFetchUsers(new AbortController());
-
-  useEffect(() => {
-    if (!userId) return;
-    if (!users.length) fetchUsers();
```

Comment:
When lots of user avatars is rendered at once `fetchUsers` is called many times, so `fetchUsers` is moved in `App.tsx` and call once. 
This implementation should be fine because lots of components is polling `fetchUsers` every few seconds